docs/spack/setup-intro.rst

```diff
@@ -0,0 +1,8 @@
+
+Spack is a generic package manager for HPC. We rely on it in the following to install
+WDMapp and its components. Setting up Spack is a one-time process on a given machine
+-- if you already have a working Spack install, you should be able to use it.
+However, in practice there are plenty of ways that things can wrong, so we provide
+tested Spack setups for a selection of machines. Following our instructions makes sure,
+that, e.g., our codes are built with compatible compilers.
```

Comment:
How about the following for the last sentence:

"Following our instructions makes sure that WDMapp is built with compatible compilers and machine-specific system packages (e.g., MPI, CUDA, etc.)."

Reply:
Thanks, that's much better.